kicad.spec.template

```diff
@@ -275,6 +293,9 @@ update-mime-database %{?fedora:-n} %{_datadir}/mime &> /dev/null || :
 
 
 %changelog
```

Comment:
I reality I think we should just remove this crud, in the modern days we have git.

Reply:
Done.  I removed the Source2 line, and all of the changelog contents.